nvflare/private/fed/server/server_commands.py

```diff
@@ -209,6 +203,7 @@ def process(self, data: Shareable, fl_ctx: FLContext):
         task_id = data.get_cookie(FLContextKey.TASK_ID)
         server_runner = fl_ctx.get_prop(FLContextKey.RUNNER)
         server_runner.process_submission(client, contribution_task_name, task_id, data, fl_ctx)
+        self.logger.info(f"submit_update process. client_name:{client.name}   task_id:{task_id}")
```

Comment:
do we want this log at info level anymore?

Reply:
same as above.